src/Components/Web.JS/src/Rendering/WebRendererInteropMethods.ts

```diff
@@ -5,32 +5,32 @@ import { DotNet } from '@microsoft/dotnet-js-interop';
 import { EventDescriptor } from './Events/EventDelegator';
 import { enableJSRootComponents, JSComponentParametersByIdentifier, JSComponentIdentifiersByInitializer } from './JSRootComponents';
 
-const interopMethodsByRenderer = new Map<number, DotNet.DotNetObject>();
+const interopMethodsByRendererId: DotNet.DotNetObject[] = [];
 
 let resolveRendererAttached : () => void;
 
 export const rendererAttached = new Promise<void>((resolve) => {
   resolveRendererAttached = resolve;
 });
 
+// TODO: Consider allowing JS initializers to run multiple times.
+// We could add a "platform" argument so the initializer can handle each
+// platform's initialization separately, if needed.
```

Comment:
Need to think about whether this is the right approach. Could we get away with keeping the existing behavior where `afterStarted()` only runs once (after the first renderer gets attached)?

Reply:
The initializer can already detect the platform by introspecting what it is being passed to it. We need to think of what pattern apps will follow when using initializers in this world, and whether we want to support doing things like having "platform" specific initializers via ".server.js" and ".browser.js".

Reply:
Yeah, this will require some more thinking. I'm going to resolve this conversation for now - I think this can be addressed in a follow-up PR.